notebooks/climatiq.ipynb

```diff
@@ -335,6 +436,48 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Industry classification estimate\n",
+    "\n",
+    "If your data comes with an industry specification code Climatiq API allows you to query emission factors, based on their NACE or ISIC industry classification codes. At the momemnt this endpoint works only with spend-based datasets, in particular EXIOBASE and GHG Protocol. \n",
+    "[More details about the industry classification estimate endpoint here](https://www.climatiq.io/docs#classifications) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_token = \"your token\" # add your Climatiq API token here\n",
```

Comment:
Maybe streamline this and call it `MY_API_KEY` here as well, like in the above cells?